new_actor.py

```diff
@@ -0,0 +1,262 @@
+import os
```

Comment:
```suggestion
#!/usr/bin/env python3

import os
```
this allows people to do ``./new_actor.py`` instead of ``python3 new_actor.py``